core/shared/utils/bh_time.c

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2023 Amazon Inc.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#include "bh_time.h"
+
+uint64
+convert_timespec_to_nanoseconds(const struct timespec *ts)
```

Comment:
How about renaming to bh_timespec_to_nanoseconds? Adding bh_ prefix is because that APIs in core/shared/utils usually start with it.